### PR TITLE
Fixed Warnings and dependencies

### PR DESCRIPTION
**Pull request by krs0**

### CODE REVIEW

File: third-upgrades/content/config/creatures/dungeon/lvl4Upg2.json [hunk 1]
```diff
@@ -105,7 +105,7 @@
 				"bonus" :
 				{
 					"type" : "SPELL_AFTER_ATTACK",
-					"subtype" : 70, // stoneGaze
+					"subtype" : "spell.stoneGaze",
 					"addInfo" : 2000 // FIXME: replace with range?
```
Comment (misiokles) [inline third-upgrades/content/config/creatures/dungeon/lvl4Upg2.json]: Very old format should be replaced with "addInfo" : [1,1]

File: third-upgrades/content/config/creatures/dungeon/lvl6Upg2.json [hunk 2]
```diff
@@ -104,15 +104,15 @@
 				"bonus" :
 				{
 					"type" : "SPELL_AFTER_ATTACK",
-					"subtype" : 81 // acidBreathDamage
+					"subtype" : "spell.acidBreath"
```
Comment (misiokles) [inline third-upgrades/content/config/creatures/dungeon/lvl6Upg2.json:107]: I guess, addInfo in this code has deafult values, no need to add this syntax (?)

File: third-upgrades/content/config/creatures/dungeon/lvl4Upg2.json (hunk 1 above)
Comment (krs0) [reply third-upgrades/content/config/creatures/dungeon/lvl4Upg2.json]: done

File: third-upgrades/content/config/creatures/dungeon/lvl6Upg2.json (hunk 2 above)
Comment (krs0) [reply third-upgrades/content/config/creatures/dungeon/lvl6Upg2.json:107]: No idea what the intention is with this ability. What level for example? There is no warning ATM so I leave it for future modders to change.

Comment (misiokles) [reply third-upgrades/content/config/creatures/dungeon/lvl6Upg2.json:107]: Like all creature-origin spells, they have 'no-level', so I'm almost sure this syntax is enough in such cases.

**Pull request merged**